src/main/java/org/openstreetmap/atlas/geography/atlas/sqlite/PackedAtlasFieldDelegator.java

```diff
@@ -0,0 +1,152 @@
+package org.openstreetmap.atlas.geography.atlas.sqlite;
+
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedTagStore;
+import org.openstreetmap.atlas.utilities.compression.IntegerDictionary;
+import org.openstreetmap.atlas.utilities.maps.LongToLongMap;
+
+/**
+ * This delegator allows the {@link SQLiteWriter} to access internal {@link PackedAtlas} fields it
+ * needs to efficiently build the database.
+ *
+ * @author lcram
+ */
+public class PackedAtlasFieldDelegator
```

Comment:
Nit. Typically they say "Delegate", not "Delegator".

Reply:
Fair enough. Honestly considering changing it to `bean` or `container` or something, since it seems to be straying a bit from the delegate pattern (since it's not delegating so much as it is simply packaging a bunch of references).